sway/ipc-server.c

```diff
@@ -1127,7 +1127,7 @@ static void ipc_event_binding(json_object *sb_obj) {
 	json_object *obj = json_object_new_object();
 	json_object_object_add(obj, "change", json_object_new_string("run"));
 	// sb_obj gets owned by the temporary json_object, too.
-	json_object_object_add(obj, "binding", json_object_get(sb_obj));
+	json_object_object_add(obj, "binding", sb_obj);
```

Comment:
`json_object_get` bumps the ref count. As a result, ownership does not actually transferr to `obj`.